barista-compose/src/androidTest/java/ComposeAssertionsTest.kt

```diff
@@ -0,0 +1,69 @@
+package com.alorma.barista_compose.assertion
+
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.test.junit4.createComposeRule
+import com.adevinta.android.barista.sample.TextComposable
+import com.alorma.barista_compose.R
+import org.junit.Rule
+import org.junit.Test
+
+class ComposeAssertionsTest {
+
+  @get:Rule
+  val composeTestRule = createComposeRule()
+
+  // assertDisplayed tests
+  @Test
+  fun assertDisplayed_StringTest() {
+    composeTestRule.setContent {
+      TextComposable(text = "Hello world")
+    }
+
+    composeTestRule.assertDisplayed(text = "Hello world")
```

Comment:
Let's use the same approach we did in the regular Barista, like `composeTestRule.assertDisplayed("Hello world")`

Reply:
yes, removed it!